app/controllers/propertyLinking/ClaimProperty.scala

```diff
@@ -73,13 +80,13 @@ class ClaimProperty @Inject()(val authenticated: AuthenticatedAction,
     }
   }
 
-  def declareCapacity(uarn: Long, address: String) = authenticated.async { implicit request =>
+  def declareCapacity(uarn: Long, address: String) = authenticatedAction { implicit request =>
     Ok(views.html.propertyLinking.declareCapacity(DeclareCapacityVM(declareCapacityForm, address, uarn)))
   }
 
-  def attemptLink(uarn: Long, address: String) = authenticated.async { implicit request =>
+  def attemptLink(uarn: Long, address: String): Action[AnyContent] = authenticatedAction.async { implicit request =>
```

Comment:
do you need .async here?

Reply:
coulda used the multi line tool here darrell.

Reply:
needed here :D